### PR TITLE
SR2024 - Update Section - Brain board

### PR DESCRIPTION
**Pull request by 8BitJosh**
Update sections on brain board

- Move the SD card flashing to the tutorials section
- Standardise on KCH naming

Created seperate issue for updating python libs and os version #496

### CODE REVIEW

File: kit/brain_board/robot_os.md [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
 ---
 
-Robot OS
-========
 
-TODO
+# Student Robotics OS
 
-# Updates
+The Raspberry Pi runs a custom operating system that contains the Student Robotics software.
```
Comment (RealOrangeOne) [inline kit/brain_board/robot_os.md:9]: Suggestion: We should specify here that it's based on ~Raspbian~ Raspberry Pi OS. The tinkerers will care.

```suggestion
The Raspberry Pi runs a custom operating system that contains the Student Robotics software. It's based on [Raspberry Pi OS](https://www.raspberrypi.com/documentation/computers/os.html) (`2023-05-03`), which in turn is based on [Debian Linux](https://www.debian.org/) (bullseye).
```

File: kit/brain_board/wifi.md [hunk 2]
```diff
@@ -3,63 +3,57 @@ layout: page
...
-Your robot has now set up its very own WiFi network! It will initially have a name starting with `robot-ZZZ` followed by some random numbers.
+Your robot has now set up its very own WiFi network!
+It will initially have a name starting with `robot-ZZZ` followed by some random numbers.
 
 You can now connect to your robot in the same way you normally connect to a WiFi network.
 You will need a WiFi key to be able to connect and you can find this inside `robot-settings.toml` on the USB drive containing your code.
 
 <div class="info">
-    If there is no <code>robot-settings.toml</code> on your USB drive, you can generate one by first running any code on your robot. For more information see our docs on <a href="{{ site.baseurl }}/kit/brain_board#robot-settings">Robot Settings</a>.
+    If there is no <code>robot-settings.toml</code> on your USB drive, you can generate one by first running any code on your robot.
```
Comment (RealOrangeOne) [inline kit/brain_board/wifi.md:23]: Nit: if you add a `markdown="1"` attribute to the `div`, you can use markdown syntax here.

File: kit/brain_board/wifi.md [hunk 3]
```diff
@@ -3,63 +3,57 @@ layout: page
...
-Changing the starting zone allows you to test how your robot handles being started in a
-different zone.
+Changing the starting zone allows you to test how your robot handles being started in a different zone.
 
-The WiFi interface will only be fully accessible when your robot is started in dev mode and will be restricted in competition mode.
-
-Changing to competition mode causes the development markers to be unreadable,
-and instead reads the competition markers, which are different. Therefore, you
-should keep your robot in development mode.
+Changing mode applies from the next time your code runs.
```
Comment (RealOrangeOne) [inline kit/brain_board/wifi.md:57]: Suggestion: Perhaps worth mentioning here that comp mode has a timeout?

File: tutorials/update_brain.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: How to update your robot
```
Comment (RealOrangeOne) [inline tutorials/update_brain.md:3]: Nit: Should this be "How to update your _brain board_"?

File: kit/brain_board/wifi.md (hunk 2 above)
Comment (8BitJosh) [reply kit/brain_board/wifi.md:23]: Really?
image

File: tutorials/update_brain.md (hunk 4 above)
Comment (8BitJosh) [reply tutorials/update_brain.md:3]: That was the original title, but it is too long and wraps wierdly

File: kit/brain_board/robot_os.md (hunk 1 above)
Comment (WillB97) [reply kit/brain_board/robot_os.md:9]: I think we only need to specify that it's based on Raspberry Pi OS, no need to go down the rabbit hole of Linux pedigrees.

File: kit/brain_board/wifi.md (hunk 2 above)
Comment (RealOrangeOne) [reply kit/brain_board/wifi.md:23]: [Apparently](https://github.com/search?q=repo%3Asrobo%2Fdocs+markdown%3D%221%22&type=code), yes. Although you may need to drop the indentation

File: tutorials/update_brain.md (hunk 4 above)
Comment (RealOrangeOne) [reply tutorials/update_brain.md:3]: Perhaps "Updating your brain board"? "Brain board updates"?

File: kit/brain_board/robot_os.md (hunk 1 above)
Comment (8BitJosh) [reply kit/brain_board/robot_os.md:9]: a4aa897

File: kit/brain_board/wifi.md (hunk 2 above)
Comment (8BitJosh) [reply kit/brain_board/wifi.md]: a4aa897

File: tutorials/update_brain.md (hunk 4 above)
Comment (8BitJosh) [reply tutorials/update_brain.md]: a4aa897

File: kit/brain_board/wifi.md (hunk 3 above)
Comment (8BitJosh) [reply kit/brain_board/wifi.md:57]: a4aa897

**Review by WillB97 — APPROVED** [review]

**Pull request merged**